lib/src/interfaces/Processors.ts

```diff
@@ -0,0 +1,9 @@
+import { Layout } from './Layout';
+
+export interface ExternalLayoutProcessor {
```

Comment:
I find the term `external` in a public facing API very confusing. If we're exporting a single type of OptionsProcessor and LayoutProcessor - the word `external` has no meaning or value to the user. Lets keep it short by removing external, WDYT?